yoyodyne/train.py

```diff
@@ -276,31 +290,34 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
     )
     parser.add_argument(
         "--train_from",
-        help="Path to ckpt checkpoint to resume training from.",
+        help="Path to ckpt checkpoint to resume training from. "
+        "Default: not enabled.",
     )
     # Other training arguments.
     parser.add_argument(
-        "--patience", type=int, help="Patience for early stopping."
+        "--every_n_epochs",
+        type=int,
+        help="Save checkpoints every n epochs. Default: not enabled.",
     )
     parser.add_argument(
-        "--save_top_k",
+        "--patience",
         type=int,
-        default=defaults.SAVE_TOP_K,
-        help="Number of checkpoints to save. Default: %(default)s.",
+        help="Patience for early stopping. Default: not enabled.",
     )
     parser.add_argument(
-        "--save_best",
-        action="store_true",
-        default=defaults.SAVE_BEST,
-        help=(
-            "Save the best checkpoint(s) according to the validation metric. "
-            "Default: True."
-        ),
+        "--top_k",
+        type=int,
+        default=defaults.TOP_K,
```

Comment:
Why do we remove the "save" prefix? I feel like top_k could be ambiguous with e.g. sampling, or decoding k hypotheses. "save_" also makes it match the ptl argument.

I would argue `save_top_k` is a better name.

Reply:
I don't really like that name (it sounds like a predicate that takes a boolean yes/no argument and not like one which takes a counting number argument) but this isn't really supposed to be a renaming-flags PR so I'll revert.

Reply:
> it sounds like a predicate that takes a boolean yes/no argument and not like one which takes a counting number argument

I see, this is a good point.